### PR TITLE
Fix #4758: Typo in _transitions.styl

### LINKED ISSUE #4758: [Bug Report] Typo in _transitions.styl
### Versions and Environment
**Vuetify:** 1.1.8
**Vue:** 2.5.16
**Browsers:** Chrome 66.0.3359.181
**OS:** Windows 10

### Steps to reproduce
Just a typo in `_transitions.styl`.

### Expected Behavior
```css
transform: translate(0, 0)
```

### Actual Behavior
```css
transtion: translate(0, 0)
```

### Reproduction Link
[https://github.com/vuetifyjs/vuetify/blob/dev/src/stylus/generic/_transitions.styl#L50](https://github.com/vuetifyjs/vuetify/blob/dev/src/stylus/generic/_transitions.styl#L50)

### PR DESCRIPTION
**Pull request by beatgrabe**
## Description
Fixes a typo in `_transitions.styl`.

## Motivation and Context
Fixes [#4758](https://github.com/vuetifyjs/vuetify/issues/4758)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have created a PR in the [documentation](https://github.com/vuetifyjs/vuetifyjs.com) with the necessary changes.

### CODE REVIEW

File: src/stylus/generic/_transitions.styl [hunk 1]
```diff
@@ -47,7 +47,7 @@
     opacity: 0
 
   &-enter-to
-    transtion: translate(0, 0)
+    transition: translate(0, 0)
```
Comment (KaelWD) [inline src/stylus/generic/_transitions.styl]: This is still invalid, looks like it's actually supposed to be `transform`. Check if this is even needed at all, it seems to work fine as is.

Comment (beatgrabe) [reply src/stylus/generic/_transitions.styl]: 😲 Oops. I will update my PR at least for sake of code consistency…

File: src/stylus/generic/_transitions.styl [hunk 2]
```diff
@@ -47,7 +47,7 @@
     opacity: 0
 
   &-enter-to
-    transtion: translate(0, 0)
+    transform: translate(0, 0)
```
Comment (johnleider) [inline src/stylus/generic/_transitions.styl]: Its behavior happens implicitly, so just remove it

Comment (beatgrabe) [reply src/stylus/generic/_transitions.styl]: Done.

**Pull request merged**